packages/server/README.md

```diff
@@ -228,14 +233,32 @@ server.on(EVENTS.POST_CREATE, (req, res, upload => {})
 
 #### `POST_RECEIVE`
 
-Called every time a `PATCH` request is handled.
+Called every time an upload has been finished writing to the store. See also
+`POST_RECEIVE_V2`.
```

Comment:
```suggestion
Called every time an upload has been finished writing to the store. This event is emitted
whenever the request handling is completed, whereas the `POST_RECEIVE_V2` event is emitted
while the request is being handled. 
```

This might explain the difference between the two events better. I am also thinking if we should choose a different name for this new event, like `DURING_RECEIVE`. It would indicate that it is emitted while the server is receiving data.

In my experience, a v2 indicator is often interpreted as a sign that this thing is new, better and should be preferred over the previous v1. Is this the case here as well? If not, we might want to drop the v2 suffix in favor of a more expressive event name.

I know that you are using these event names to be similar to tusd's hooks, but our decision to use `post-receive` there is also a bit questionable in hindsight. So we don't have to restrict ourselves to whatever odd name we came up with in tusd.

Reply:
Ok, then let's leave it like this.